charactersheet/charactersheet/index.html

```diff
@@ -216,14 +216,15 @@ <h4 class="modal-title" id="exportModalLabel" data-dismiss="modal">Export</h4>
           data-dismiss="modal">Create a new character</h4>
       </div>
       <div class="modal-body">
-        <p>If you play more than one character per game, or if you
-        play multiple games, you're in luck! You can have
-        multiple characters and switch between them at any time!</p>
+        <p>If you play more than one character per game,
```

Comment:
It's not just `per game` it's multiple characters in general.
Can we change it to more of this:
> If you play more than one character, run multiple campaigns...